lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -4,6 +4,19 @@
 
 module KubernetesDeploy
   class KubernetesResource
+    def self.logger=(value)
+      @logger = value
+    end
+
+    def self.logger
+      @logger ||= begin
+        l = Logger.new($stderr)
+        l.formatter = proc do |_severity, _datetime, _progname, msg|
+          "#{msg}\n"
```

Comment:
Yeah, double timestamps is not desirable, but why are there already double timestamps on that `ERROR`? I'd still argue we should have timestamp on `KUBESTATUS` because it's an implementation detail that it only happens at a defined interval (e.g. I didn't even know that).